### PR TITLE
Add app home

### PR DESCRIPTION
**Pull request by JustinHsie**
Created app homepage with react-admin, the link is in the wiki

It took me awhile to learn how the framework works - is this too much framework? ie, should I be building more components by hand?

I followed their tutorial to get set up and it included API calls to jsonPlaceholder. Those became the automatically generated User and Posts links, we can customize with our actual API calls later

I added the custom navigation links as well (Home, Search, New List, New Place)

### CODE REVIEW

**Comment by Exegetech** [comment]
See my reply on the issue https://github.com/JustinHsie/eats/issues/8

**Comment by Exegetech** [comment]
@JustinHsie Looks like this, is this correct?
Screen Shot 2021-01-12 at 11 14 59 AM

File: client/src/components/Lists.js
Comment (Exegetech) [inline client/src/components/Lists.js:7]: Seems like this can safely be removed.

File: client/src/components/Lists.js
Comment (Exegetech) [inline client/src/components/Lists.js]: export default these days are considered an anti-pattern. It also will break compatibility with Babel/TypeScript later.  Use `export const List` in line 5

File: client/src/components/Menu.js
Comment (Exegetech) [inline client/src/components/Menu.js]: Same as above

File: client/src/fakeData/lists.js
Comment (Exegetech) [inline client/src/fakeData/lists.js]: Same as above

File: client/src/styles/App.css
Comment (Exegetech) [inline client/src/styles/App.css:1]: I'm curious what style of naming convention for CSS did you learn? Just want to make sure it is consistent later. I usually use BEM but feel free to use whatever you want, as long as consistent.

**Comment by JustinHsie** [comment]
Yeah! I know it doesn't look great, but is this ok for now? Yesterday I left some comments on the 'issues' as a summary of my thoughts. Things might change as I go along, but is there anything else I should add?
And the later pull requests should have links to the other pages, and the heroku site should have the most updated version

**Review by Exegetech — CHANGES_REQUESTED** [review]
> Yeah! I know it doesn't look great, but is this ok for now? Yesterday I left some comments on the 'issues' as a summary of my thoughts. Things might change as I go along, but is there anything else I should add?
> And the later pull requests should have links to the other pages, and the heroku site should have the most updated version

Alright once you address my change I'll merge it. Don't merge it yet.

File: client/src/styles/App.css
Comment (JustinHsie) [reply client/src/styles/App.css:1]: I'm actually not sure if I explicitly learned a naming convention, but I'll start using BEM now

**Comment by JustinHsie** [comment]
Ok I think I made the changes you requested? I was a little confused because I saw your comments under Files Changed but it also showed changes I previously made to package.json and App. To clarify, those weren't your requests, right? Just stuff from the last commit?

**Comment by Exegetech** [comment]
Yeah this is fine. Now usually if you work in another person/organization repo, this Merge pull request will be blocked before 1 or 2 reviews and passing CI system. But since this is your own repo you can merge it yourself. Pretend that you can't now, so only I can merge it.

When you merge, you have the choice to just merge like that, so any commits that you build in this branch will be merged just like that to master. I don't like that approach because it is not clean. When a developer work in a branch, they usually commit for the sake of saving work, regardless what the actual app state (broken or not) at that commit. Commit needs to be atomic, meaning, if I revert a commit, I need to be able to build the app and the app must work and not broken. Imagine if during release you introduce a bug, and your boss tells you to revert back the commit. You want to revert the commit and not finding the app in a half broken state.

Therefore a better approach before merging to master is to squash these commits in this PR. On the green button below, there is a dropdown, there will be an option that says "Squash" and merge. Choose that, and then you will be prompted to input a description. Make a good description. 

What I usually do, in the title I put the feature or branch name (if you use ticketing system, the ticket number), along with the issue number in parentheses. In the body I describe what changes I made.

For example

```
app-home(#11)

Changes:
- Do x
- Do y
- Do z

Close #8
```

**Comment by Exegetech** [comment]
I have squashed and merged, now check `master` branch and see the commit history. It looks clean like that, and atomic.

**Comment by JustinHsie** [comment]
Awesome, that's really valuable to know. Thank you so much for the git lesson! I'll do the cherry-picking for the other branch now

**Pull request merged**
